src/Schema/SchemaValidator.php

```diff
@@ -126,7 +127,7 @@ public function validate($data, CebeSchema $schema, ?BreadCrumb $breadCrumb = nu
                 (new Items($schema, $this->validationStrategy, $breadCrumb))->validate($data, $schema->items);
             }
 
-            if ($schema->type === CebeType::OBJECT) {
+            if ($schema->type === CebeType::OBJECT || (isset($schema->properties) && is_array($data))) {
```

Comment:
I hope we should check for assoc arrays here. There is an util for this 
https://github.com/thephpleague/openapi-psr7-validator/blob/master/src/Foundation/ArrayHelper.php#L23

Reply:
Added :+1: